src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -11,7 +11,7 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 	const { issueKey, tenantUrl } = req.query;
 	const jiraHostQuery = req.query.jiraHost as string;
 
-	if (!tenantUrl && !jiraHostQuery) {
+	if (!tenantUrl && !jiraHostQuery && res.locals.jiraHost) {
```

Comment:
Post flag 100% this will become
`if (res.locals.jiraHost) {`

Reply:
`&& res.locals.jiraHost` <-- So the code throw an error if there's a `jiraHost` in the, say, cookie?